Resources/Private/Translations/en/Main.xlf

```diff
@@ -380,6 +380,12 @@
             <trans-unit id="copyNodeTypeNameToClipboard" xml:space="preserve">
                 <source>Copy node type to clipboard</source>
             </trans-unit>
+            <trans-unit id="dimensions.doesNotExistsInDimension" xml:space="preserve">
+                <source>The current document does not exist in the dimension</source>
```

Comment:
I might miss the context, but I was wondering: What is "the dimension"? Would something like "...does not exist in the currently active content dimension" be clearer?